src/table-filter-label.js

```diff
@@ -1,13 +1,9 @@
-import React, { useState, useRef } from 'react'
+import React, { useRef } from 'react'
 import PropTypes from 'prop-types'
 
-import ButtonBase from '@material-ui/core/ButtonBase'
-import Popper from '@material-ui/core/Popper'
-import Grow from '@material-ui/core/Grow'
-import Paper from '@material-ui/core/Paper'
-import ClickAwayListener from '@material-ui/core/ClickAwayListener'
 import FilterListRoundedIcon from '@material-ui/icons/FilterListRounded'
-import { makeStyles } from '@material-ui/core/styles'
+
+import { BaseComponents, makeStyles } from '@eqworks/lumen-labs'
```

Comment:
refine the import for specific base components for easy tree-shaking by properly configured transpiling tools:

```js
import { BaseComponents: { ButtonBase, DialogBase }, makeStyles } ...
```

Reply:
react doesn't let me to apply this type of import is there anything I missed?
<img width="811" alt="Screen Shot 2022-03-16 at 3 06 11 PM" src="https://user-images.githubusercontent.com/46459367/158667769-5b00b4a4-7e16-47b0-a2da-a0ac512eb22f.png">

Reply:
perhaps something like `import ButtonBase from '@eqworks/lumen-labs/dist/base-components/button-base'`